tests/conftest.py

```diff
@@ -0,0 +1,125 @@
+from jinja2 import Environment
+from logging import basicConfig, getLogger, INFO
+from OpenSSL import crypto
+from os import path
+import pytest
+from subprocess import (
+    check_output,
+    Popen,
+    STDOUT,
+    SubprocessError,
+    TimeoutExpired,
+)
+from time import sleep
+
+
+pytest_plugins = [
+    "mock_fixtures",
+]
+
+
+def pytest_addoption(parser):
+    parser.addoption("--nginx", default="../nginx/objs/nginx")
+    parser.addoption("--module", default="build/ngx_otel_module.so")
+    parser.addoption("--showlog", action="store_true")
+    parser.addoption("--otelcol")
```

Comment:
Looks like, we'll also need "--globals" option, mostly for mgmt/license_token stuff.

Reply:
OK, added.